server/src/api/merchants/index.ts

```diff
@@ -24,9 +24,19 @@ import {merchantsService} from 'services';
 
 const router: express.Router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  const merchants = (await merchantsService.getMerchants())[0];
-  res.send(merchants);
-});
+router.get(
+  '/',
+  async (
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ) => {
+    try {
+      const merchants = (await merchantsService.getMerchants())[0];
```

Comment:
Why you don't set res anymore?

Reply:
ohh thanks for noticing! I accidentally removed the `res.send()`